clang/lib/DPCT/GenMakefile.cpp

```diff
@@ -286,18 +286,32 @@ static void getCompileInfo(
         IsSystemInclude = true;
       } else if (llvm::StringRef(Option).starts_with("-D")) {
         // Parse macros defined.
+        bool NeedSigleQuote = false;
```

Comment:
```suggestion
        bool NeedSingleQuote = false;
```

Reply:
Good catch, will update it in the next PR.